src/services/assetService.ts

```diff
@@ -84,6 +95,15 @@ export class AssetService {
         }
     }
 
+    public static getMimeType = async (uri: string) => {
+        const FileType = require("file-type/browser");
+        const response = await fetch(uri);
```

Comment:
We don't want to fetch each document twice

Reply:
not anymore